app/src/pages/projects/ProjectsPage.tsx

```diff
@@ -63,13 +76,11 @@ export function ProjectsPageContent({
 
   const data = useLazyLoadQuery<ProjectsPageQuery>(
     graphql`
-      query ProjectsPageQuery($timeRange: TimeRange!) {
+      query ProjectsPageQuery {
```

Comment:
technically since this is can now be pushed into a loader now since it never needs to refetch